server/views/pages/alerts.njk

```diff
@@ -148,6 +148,7 @@
   </form>
   
   <p id="lastUpdated"></p>
+  <div id="alertsErrorStatus"></div>
```

Comment:
Again re my suggestion, could just use the proper `mojAlert` macro here and have it hidden?